index.html

```diff
@@ -192,5 +192,93 @@ <h2 class="card-title">profesional art printing date</h2>
         </div>
       </div>
     </section>
+    <section class="about-section">
+      <div class="about-header">
+        <h1>About me</h1>
+        <p class="about-header-text">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <p class="btn-about">
+          <span class="btn-about-text">get my resume</span>
+        </p>
+        <img
+          class="about-bg-img"
+          src="./images/about-bg.png"
+          alt="about background image"
+        />
+        <div class="about-content-divider">jjhg</div>
```

Comment:
- [ ] (OPTIONAL) You can use an `hr` tag here to make the horizontal line and give the `hr` a border style color of white.